src/diffusers/models/attention_processor.py

```diff
@@ -378,7 +378,7 @@ def set_processor(self, processor: "AttnProcessor", _remove_lora: bool = False)
             _remove_lora (`bool`, *optional*, defaults to `False`):
                 Set to `True` to remove LoRA layers from the model.
         """
-        if hasattr(self, "processor") and _remove_lora and self.to_q.lora_layer is not None:
+        if not USE_PEFT_BACKEND and hasattr(self, "processor") and _remove_lora and self.to_q.lora_layer is not None:
```

Comment:
@patrickvonplaten with this change the test for `self.to_q.lora_layer` is not needed since when `USE_PEFT_BACKEND = True`, the `self.to_q` will be `nn.Linear` and one will get an error

I had the same to use `USE_PEFT_BACKEND` for the wuerstchen prior pipeline see #5704

Reply:
Could you add that change in #5704?

Reply:
i did